training/app_init.py

```diff
@@ -13,3 +24,24 @@ def main():
 
 if __name__ == '__main__':
     main()
+
+
+
+
+
+
+from flask_admin import Admin
+from flask_admin.contrib.sqla import ModelView
+from training.models.users import User
```

Comment:
imports should be on top. If this is a problem try splitting this code into several modules

Reply:
It will be fixed in the next PR in the next branch.